example-tests/src/test/java/com/epam/restassured/BasicServiceTest.java

```diff
@@ -7,59 +7,104 @@
 //TODO: 5. create DDT script for REST script
 //TODO: 6. create DDT script for webdriver script
 //TODO: 7. create rest script without BDD style (using JUnit asssertions)
+
+import com.epam.restassured.pageobjects.HomePageObject;
+import com.epam.restassured.pageobjects.ThankYouPageObject;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import java.util.ArrayList;
+import java.util.List;
+
 import static com.jayway.restassured.RestAssured.given;
 import static com.jayway.restassured.RestAssured.when;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.is;
 
-import java.util.ArrayList;
-import java.util.List;
+/**
+ * Created by Tamas_Csako
+ */
+public class BasicServiceTest {
+    private static Logger log = Logger.getLogger(BasicServiceTest.class.getName());
+    private static final int NUMBER_OF_RESPONSE = 1;
+    private static final String CONTENT_NUMBER_OF_ELEMENTS = "numberOfElements";
```

Comment:
remove these constants and use "numberOfElements", etc. in the rest verification